README.md

```diff
@@ -269,3 +270,23 @@ The default configuration is the following:
 - There's a defined `whitelist` where all reducers that want to be persisted must be declared.
 - The `storage` engine is `AsyncStorage` but you can change it if needed, for example: https://github.com/CodingZeal/redux-persist-sensitive-storage if you need keychan storage on iOS.
 - If you ever need to set up migrations to keep your reducers up to date, please check [this link](https://github.com/rt2zz/redux-persist#migrations).
+
+### Adding a Splash Screen
+
+It was used the package [react-native-splash-screen](https://github.com/crazycodeboy/react-native-splash-screen) to add the splash screen functionality with placeholder images.
```

Comment:
```suggestion
The package used to add the splash screen functionality was [react-native-splash-screen](https://github.com/crazycodeboy/react-native-splash-screen) and uses placeholder images.
```

Maybe something like this? I feel a bit weird the `It was used the package` sentence.

Reply:
Totally agree